docs/cli.rst

```diff
@@ -1284,7 +1290,8 @@ Update the nextflow runtime to the latest available version.
 
 **Description**
 
-The ``self-update`` command directs the ``nextflow`` cli to update itself to the latest stable release.
+The ``self-update`` command directs the ``nextflow`` cli to update itself to the latest stable release. 
+This command also updates the `Nextflow launcher script <https://github.com/nextflow-io/nextflow/blob/master/nextflow>`_ in addition to the Java artifacts, whereas the ``NXF_VER`` environment variables only updates the Java artifacts.
```

Comment:
Don't think this is correct. The `self-update` is the same of `-self-update` and they do *not* update the launcher

Reply:
Whoops!

@pditommaso , then let me cofirm my understanding here

1. `-self-update` updates only java artifacts
2. `NXF_VER` updates java artifacts and the launcher script

Therefore when `NXF_EDGE=1 nextflow -self-update`  is used, only the java artifacts are updated to the latest edge version.

Is this correct?

Reply:
Actually I was wrong here. `self-update` and `-self-update` update both nextflow runtime and the launcher script. 


`NXF_VER` is meant to enforce the use of a specific nf version. Don't think it should be mentioned in this context.